core/dive.c

```diff
@@ -749,10 +749,9 @@ struct sample *prepare_sample(struct divecomputer *dc)
 
 		// Copy the sensor numbers - but not the pressure values
 		// from the previous sample if any.
-		if (nr) {
-			sample->sensor[0] = sample[-1].sensor[0];
-			sample->sensor[1] = sample[-1].sensor[1];
-		}
+		if (nr)
```

Comment:
i think we should wrap the code with brackets here:
```
if (nr) {
    for (int idx = 0; idx < MAX_SENSORS; idx++)
        sample->sensor[idx] = sample[-1].sensor[idx];
}
```

i'm not sure that we a consistent about this, but my logic here is that the `for` loop wraps a single line and there is no need for `{ }`. the `if (nr)` branch on the other hands wraps more than one line, so `{ }` should be there.